app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    file_name1 = command[1]
+    file_name2 = command[2]
+    if file_name1 == file_name2 and command[0] == "cp":
+        return
+    with open(file_name1, "r") as file_in, open(file_name2, "w") as file_out:
+        for line in file_in.readlines():
```

Comment:
You can do without the readlines() method, since in large projects it will take up a lot of memory; a simple loop for line in file_in is enough for line-by-line search

Reply:
@VitalyBashkiser is right, try to do this in one line without loop